.stylelintrc

```diff
@@ -1,12 +1,12 @@
 {
   "extends": ["stylelint-config-vaadin"],
-  "plugins": ["./custom-linter-rules/no-shorthand-with-unresolved-longhand.js"],
+  "plugins": ["./custom-stylelint-rules/no-shorthand-with-unresolved-longhand.js"],
```

Comment:
I would suggest some common top level folder e.g. `rules` with `rules/eslint` and `rules/stylelint` subfolders.

Reply:
Done.